CMakeLists.txt

```diff
@@ -3,12 +3,23 @@ cmake_minimum_required(VERSION 3.16.8)
 project(hipify-clang)
 
 include(GNUInstallDirs)
-find_package(LLVM REQUIRED CONFIG PATHS ${CMAKE_PREFIX_PATH})
 
-message(STATUS "Found LLVM ${LLVM_PACKAGE_VERSION}:")
-message(STATUS "   - CMake module path: ${LLVM_CMAKE_DIR}")
-message(STATUS "   - Include path     : ${LLVM_INCLUDE_DIRS}")
-message(STATUS "   - Binary path      : ${LLVM_TOOLS_BINARY_DIR}")
+option(HIPIFY_INCLUDE_IN_HIP_SDK "Include HIPIFY in HIP SDK" OFF)
+
+if(HIPIFY_INCLUDE_IN_HIP_SDK)
+  if(NOT WIN32)
```

Comment:
It doesn't exclude `MSVC`. Should we strict it more (like below) or do we want to support also `-G "Visual Studio XX 20YY"` for `HIPIFY_INCLUDE_IN_HIP_SDK` set to `ON`?

```cmake
if(HIPIFY_INCLUDE_IN_HIP_SDK)
  if(NOT WIN32)
    message(FATAL_ERROR "HIPIFY_INCLUDE_IN_HIP_SDK is only supported on Windows")
  elseif(MSVC)
    message(FATAL_ERROR "HIPIFY_INCLUDE_IN_HIP_SDK is targeting Ninja only")
  endif()
endif()
```

Reply:
MSVC is a compiler, not a build system, your code snippet above will always hit the "HIPIFY_INCLUDE_IN_HIP_SDK is targeting Ninja only" error with HIPIFY_INCLUDE_IN_HIP_SDK=ON, because we have to use MSVC from the DK.

I can restrict it to Ninja, but I am not sure if it is the right choice. For example, someone could use Unix Makefiles.

I think it would be better to check that it is not Visual Studio, like in the following code snippet:

`
if(CMAKE_GENERATOR MATCHES "Visual Studio")
  message(FATAL_ERROR "...")
endif()
`

Reply:
That's better and enough, I think. Thanks!